src/components/views/articles/SingleArticle.js

```diff
@@ -0,0 +1,235 @@
+/* eslint-disable camelcase */
```

Comment:
Please place the exceptions inline with the code you are trying to exempt.

Reply:
This has been done. Thanks @bettblake08